Formula/ld-find-code-refs.rb

```diff
@@ -0,0 +1,16 @@
+# Documentation: https://docs.brew.sh/Formula-Cookbook
+#                https://www.rubydoc.info/github/Homebrew/brew/master/Formula
+class LdFindCodeRefs < Formula
+  desc "Enables you to keep track of feature flag usage in your code from LaunchDarkly."
+  homepage "https://launchdarkly.com/"
+  url "https://github.com/launchdarkly/ld-find-code-refs/releases/download/0.2.0/ld-find-code-refs_0.2.0_darwin_amd64.tar.gz"
+  sha256 "bc900537a491d40f783bb0dd81ab5373682683e49e41af4bf88014eb4067a882"
+  version "0.2.0"
+
+  depends_on "ag"
```

Comment:
isn't this `the_silver_searcher`? https://github.com/ggreer/the_silver_searcher#installing

Reply:
![screen shot 2019-01-17 at 9 42 46 am](https://user-images.githubusercontent.com/1269308/51337550-4507a680-1a3c-11e9-9082-fd9f63cef34e.png)